velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -133,4 +112,57 @@ class HiveConnectorTestBase : public OperatorTestBase {
   std::unique_ptr<folly::IOThreadPoolExecutor> executor_;
 };
 
+class HiveConnectorSplitBuilder {
+ public:
+  HiveConnectorSplitBuilder(std::string filePath)
+      : filePath_{std::move(filePath)} {}
+
+  HiveConnectorSplitBuilder& start(uint64_t start) {
+    start_ = start;
+    return *this;
+  }
+
+  HiveConnectorSplitBuilder& length(uint64_t length) {
+    length_ = length;
+    return *this;
+  }
+
+  HiveConnectorSplitBuilder& fileFormat(dwio::common::FileFormat format) {
+    fileFormat_ = format;
+    return *this;
+  }
+
+  HiveConnectorSplitBuilder& partitionKey(
+      std::string name,
+      std::optional<std::string> value) {
+    partitionKeys_.emplace(std::move(name), std::move(value));
+    return *this;
+  }
+
+  HiveConnectorSplitBuilder& tableBucketNumber(int32_t bucket) {
+    tableBucketNumber_ = bucket;
+    return *this;
+  }
+
+  std::shared_ptr<connector::hive::HiveConnectorSplit> build() const {
+    return std::make_shared<connector::hive::HiveConnectorSplit>(
+        kHiveConnectorId,
+        "file:" + filePath_,
+        fileFormat_,
+        start_,
+        length_,
+        partitionKeys_,
+        tableBucketNumber_);
+  }
+
+ private:
+  const std::string connectorId_;
```

Comment:
nit: Is this being used ?

Reply:
Good catch. This is not used. Let me remove.